compiler/common-artifacts/CMakeLists.txt

```diff
@@ -274,13 +281,19 @@ foreach(RECIPE IN ITEMS ${RECIPES})
     )
     list(APPEND TEST_DEPS ${TC_DIRECTORY})
 
+    set(ADDITIONAL_OPTIONS )
```

Comment:
Is it intended?
```suggestion
    set(ADDITIONAL_OPTIONS)
```

Reply:
Ah, yes... Just to set `ADDITIONAL_OPTIONS` as empty string.

Reply:
maybe add a note?

Reply:
added a comment;

Reply:
(optinal) Maybe empty string can be created like below.
```cmake
set(ADDITIONAL_OPTIONS "")
Or
set(ADDITIONAL_OPTIONS)
```